### PR TITLE
Artifact database

### PR DESCRIPTION
**Pull request by rjmacleod**
- Some CSS changes that got dragged into this branch somehow
- Add /assets/ folder to backend, populate with images
- Modifiy dockerfile to copy /assets
- Add "IMAGE_DIR" property to Configuration.py
- Add all artifacts with completed fields to db.json
- Create ImageFetcher.py flask api route

### CODE REVIEW

File: backend/src/WebServer/controllers/images/ImageFetcher.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from src.WebServer.WebServerInit import WebServerInit
+from src.WebServer.decorators.HTTPLogger import http_logger
+from src.Configuration import Configuration, CONF_INSTANCE
+
+from flask import Flask, request, send_from_directory, abort
+flask_ref: Flask = WebServerInit.flask
+
+class RenderHTML:
```
Comment (xxdunedainxx) [inline backend/src/WebServer/controllers/images/ImageFetcher.py:8]: Lol change this class to something else like ImageController or something

File: backend/src/WebServer/controllers/images/ImageFetcher.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  @staticmethod
+  @flask_ref.route('/get-image/<id:image_name>')
+  @http_logger
+  def get_image(image_name):
+    try:
+      return send_from_directory(CONF_INSTANCE.IMAGE_DIR, image_name)
+    except FileNotFoundError:
+      abort(404)
```
Comment (xxdunedainxx) [inline backend/src/WebServer/controllers/images/ImageFetcher.py:17]: `return "failed",404` ??

File: backend/db.json [hunk 3]
```diff
@@ -10,40 +10,168 @@
...
       },
       {
         "ArtifactName" : ["California Moray Eel", "Moray Eel"],
         "Clue": "This ocean dwelling animal is unlike other fish, it only has one singular continuous fin and does not have gill covers, so to breath it must constantly open and close its mouth.",
         "AdditionalHint" : "This ocean dwelling animal is a long, slim snakelike fish.",
         "CorrectMessage" : "California Moray Eels have a second set of jaws in their throat that extend forward to grab prey with backwards facing teeth that drag the prey back into the throat. This second set of jaws is called pharyngeal jaws. They use this feeding method to mainly eat fish, but also feed on octopus, lobsters and shrimp in the rocky reefs of southern California.",
-        "MediaLink" : "https://upload.wikimedia.org/wikipedia/commons/thumb/3/3f/California_Moray_Eel_%2832582219365%29.jpg/800px-California_Moray_Eel_%2832582219365%29.jpg",
-        "Credit" : ""
+        "MediaLink" : "https://lindsay-wildlife-apps.zee-aws.net/get-image/1.png",
```
Comment (xxdunedainxx) [inline backend/db.json:21]:
```suggestion
        "MediaLink" : "https://lindsay-wildlife-apps.zee-aws.net/backend/get-image/1.png",
```

I think these should look like this

Comment (rjmacleod) [reply backend/db.json]: Fixed!

**Review by xxdunedainxx — APPROVED** [review]
LGTM 👍

File: backend/src/MultiThreading/ExitHandlers.py [hunk 4]
```diff
@@ -6,6 +6,10 @@
 from src.util.ErrorFactory import errorStackTrace
 from src.util.FileIO import FileIO
 
+SIGNALS_TO_IGNORE : [int] =[
```
Comment (rjmacleod) [inline backend/src/MultiThreading/ExitHandlers.py:9]: @xxdunedainxx I can remove these changes to ExitHandlers.py from this PR, correct?

Comment (xxdunedainxx) [reply backend/src/MultiThreading/ExitHandlers.py:9]: yah this shouldn't be in this PR, but can be file a github issue to add this into ExitHandlers in a separate PR?

**Pull request closed without merging**